packager/media/codecs/aac_audio_specific_config.cc

```diff
@@ -137,27 +137,28 @@ bool AACAudioSpecificConfig::Parse(const std::vector<uint8_t>& data) {
          channel_config_ <= 7;
 }
 
-bool AACAudioSpecificConfig::ConvertToADTS(std::vector<uint8_t>* buffer) const {
-  size_t size = buffer->size() + kADTSHeaderSize;
-
+bool AACAudioSpecificConfig::ConvertToADTS(std::vector<uint8_t>& audio_frame, const uint8_t* data, size_t data_size) const {
```

Comment:
Can you change the function signature to:
bool AACAudioSpecificConfig::ConvertToADTS(const uint8_t* data, size_t data_size,
                                                                            std::vector<uint8_t>* audio_frame) const {

Google C++ style guide requires the output parameter after the input parameter. It also requires output to be a pointer. See https://google.github.io/styleguide/cppguide.html#Output_Parameters for details.

Can you also break it into two lines and make sure there are a maximum of 80 characters per line?